recipe/meta.yaml

```
@@ -22,7 +24,19 @@ requirements:
 
 test:
   commands:
+    - if [ ! -d ${PREFIX}/lib ]; then mkdir -p ${PREFIX}/lib; fi  # [linux or osx]
+    - cp ./sobin/libgs.so* ${PREFIX}/lib/  # [linux]
+    - cp ./sobin/libgs.dylib* ${PREFIX}/lib/  # [osx]
+    - if [ ! -f ${PREFIX}/lib/libgs.so ]; then ln -s ${PREFIX}/lib/libgs.so.{{ major }} ${PREFIX}/lib/libgs.so; fi  # [linux]
+    - if [ ! -f ${PREFIX}/lib/libgs.dylib ]; then ln -s ${PREFIX}/lib/libgs.dylib.{{ major }} ${PREFIX}/lib/libgs.dylib; fi  # [osx]
     - gs --version
+    - test -f ${PREFIX}/lib/libgs.so.{{ major }}  # [linux]
```

You can use this variable:
```suggestion
    - test -f ${PREFIX}/lib/libgs${SHLIB_EXT}.{{ major }}  # [unix]
```

---

Awesome, thanks!